web-ui/src/main/resources/catalog/components/viewer/ViewerDirective.js

```diff
@@ -71,26 +71,47 @@
               };
               scope.ol3d = null;
 
+
               // 3D mode is allowed and disabled by default
               scope.is3DModeAllowed = gnConfig['map.is3DModeAllowed'] || false;
               scope.is3dEnabled = gnConfig['is3dEnabled'] || false;
 
-
               // Synch only background layer between main map and search map
               scope.synAllLayers = false;
+
               scope.map.getLayers().on('add', function() {
                 if (angular.isDefined(gnSearchSettings.searchMap)) {
+
                   var layers = gnSearchSettings.searchMap.getLayers();
+
                   scope.map.getLayers().forEach(function(l) {
                     if (scope.synAllLayers === false &&
                         l.get('group') === 'Background layers') {
                       layers.removeAt(0);
                       layers.insertAt(0, l);
                     }
                     // TODO: Synchronize all layers
+
                   });
+
                 }
               });
+
+              scope.syncMod = function(map) {
+
+                scope.synAllLayers=!scope.synAllLayers;
+
+                var layers = gnSearchSettings.searchMap.getLayers();
+                layers.clear();
```

Comment:
If you refer to the bboxes on the mini map, they are not being dropped, and they don't show up in the layer count of the mini map either...  

![bbox](https://cloud.githubusercontent.com/assets/1038897/17395049/9785065a-5a2c-11e6-9c98-e2826724cc61.png)